exposed-core/src/main/kotlin/org/jetbrains/exposed/sql/statements/UpsertStatement.kt

```diff
@@ -30,4 +30,17 @@ open class UpsertStatement<Key : Any>(
         }
         return functionProvider.upsert(table, arguments!!.first(), onUpdate, where, transaction, keys = keys)
     }
+
+    override fun arguments(): List<Iterable<Pair<IColumnType, Any?>>> {
+        return arguments!!.map { args ->
```

Comment:
should we use `?.` with `?: emptyList()` instead?

Reply:
Sure, I'll change here and in the original super `InsertStatement`.